tests/testfiles/CommonMark_0.30/entity_and_numeric_character_references/001.test

```diff
@@ -1,18 +1,19 @@
 %   ---RESULT--- "example": 25,
 %   
-%   <p>  &amp; © Æ Ď
-%   ¾ ℋ ⅆ
-%   ∲ ≧̸</p>
+%   <p><em>  &amp; © Æ Ď</em>
+%   <em>¾ ℋ ⅆ</em>
+%   <em>∲ ≧̸<</em>/p>
 %   
 %   ---\RESULT---
 
 \def\markdownOptionHtml{true}
 <<<
-&nbsp; &amp; &copy; &AElig; &Dcaron;
-&frac34; &HilbertSpace; &DifferentialD;
-&ClockwiseContourIntegral; &ngE;
+*&nbsp; &amp; &copy; &AElig; &Dcaron;*
+*&frac34; &HilbertSpace; &DifferentialD;*
+*&ClockwiseContourIntegral; &ngE;*
 >>>
 documentBegin
-nbsp
-ampersand
+emphasis: (nbsp) (ampersand) © Æ Ď
+emphasis: ¾ ℋ ⅆ
+emphasis: ∲ ≧̸
 documentEnd
```

Comment:
For technical reasons, top-level text nodes are not part of the output. In tests of block elements, this is not a problem and can even improve readability in some cases. However, in tests of inline elements, it is often useful to surround the test case with an inline element such as emphasis, so that we can see (and test) the top-level text nodes in the output as well. Same for tests 001 through 006, 013, and 016.